src/geometries/primitives/point.jl

```diff
@@ -80,12 +80,19 @@ Return the coordinates of the `point`.
 """
 coords(A::Point) = A.coords
 
+"""
+    cartvalues(point)
+
+Return the cartesian coordinate values of the `point`.
+"""
+cartvalues(A::Point) = CoordRefSystems.values(convert(Cartesian, A.coords))
+
```

Comment:
this function is a util, not part of the Point api, it should be moved elsewhere